ExampleMod/Content/Items/Armor/Vanity/ExampleRobe.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Terraria;
+using Terraria.ModLoader;
+using Terraria.ID;
+
+namespace ExampleMod.Content.Items.Armor.Vanity
+{
+	// See also: ExampleCostume
+	[AutoloadEquip(EquipType.Body)]
+	internal class ExampleRobe : ModItem
```

Comment:
In the 1.3 ExampleRobe the class was `internal`, I'm not sure why this is necessary, but I included it in the port just in case.